README.md

```diff
@@ -159,13 +174,75 @@ To uninstall/delete a deployment with name `my-splunk-otel-collector`:
+
+You can add additional log files to be ingested from Kubernetes host machines and kubernetes volumes by configuring extraHostPathMounts and extraHostFileConfig in the values.yaml file used to deploy Splunk OpenTelemetry Collector for Kubernetes.
+
+[Example](https://github.com/splunk/sck-otel/blob/main/example/values/extraHostFileValues.yaml#L102)
+
+### Override underlying OpenTelemetry Agent configuration
+If you want to use your own OpenTelemetry Agent configuration, you can build a [OpenTelemetry Agent config](https://github.com/splunk/sck-otel/blob/main/example/manifests/otel_config.yaml) and override our default config by configuring configOverride in the values.yaml file used to deploy Splunk OpenTelemetry Collector for Kubernetes.
+
+### Adding Audit logs from Kubernetes host machines
+You can ingest audit logs from your Kubernetes cluster by configuring extraHostPathMounts and extraHostFileConfig in the values.yaml file used to deploy Splunk OpenTelemetry Collector for Kubernetes.
+
+[Example](https://github.com/splunk/sck-otel/blob/main/charts/sck-otel/values.yaml#L122)
+
+### Processing Multi-Line Logs
+Splunk Connect for Kubernetes-OpenTelmetry supports parsing of multiline logs to help read, understand and troubleshoot the multiline logs in a better way.
+Process multiline logs by configuring `multilineSupportConfig` section in values.yaml.
+
+[Example](https://github.com/splunk/sck-otel/blob/9bd92b9b2054b85eadfd744888cc19ebb46b0081/charts/sck-otel/values.yaml#L77)
+
+If you have a specific format you are using for formatting a python stack traces, you can take an example of your stack trace output and use https://regex101.com/  to find a golang regex that works for your format and specify it in the config file for the config option "first_entry_regex" and for the config option pass in the appropriate container name.
+
+### Tweak Performance/resources used by Splunk OpenTelemetry Collector for Kubernetes
+If you want to tweak performance/cpu and memory resources used by  Splunk OpenTelemetry Collector for Kubernetes change the available cpu and memory for the Opentelemtry Agent by configuring resources:limits:cpu and resources:limits:memory in the values.yaml file used to deploy Splunk OpenTelemetry Collector for Kubernetes.
+
+[Example](https://github.com/splunk/sck-otel/blob/main/charts/sck-otel/values.yaml#L143)
+
+## Maintenance And Support
+Splunk OpenTelemetry Collector for Kubernetes is supported through Splunk Support assuming the customer has a current Splunk support entitlement ([Splunk Support](https://www.splunk.com/en_us/about-splunk/contact-us.html#tabs/tab_parsys_tabs_CustomerSupport_4)). For customers that do not have a current Splunk support entitlement, please search [open and closed issues](https://github.com/splunk/sck-otel/issues?q=is%3Aissue) and create a new issue if not already there.
+The current maintainers of this project are the DataEdge team at Splunk.
```

Comment:
This is already covered in troubleshooting doc. I think it can be removed, or need to be updated to cover both Observability and Enterprise